src/agent/onefuzz-supervisor/src/config.rs

```diff
@@ -47,6 +49,8 @@ struct RawStaticConfig {
 
     pub onefuzz_url: Url,
 
+    pub multi_tenant_domain: Option<String>,
```

Comment:
Is there a default value for this public? 

Might consider changing this to just 'domain' instead of 'multi_tenant_domain'

Reply:
If no value is provided, `multi_tenant_domai` would be `None`. 

I dont have a preference for name but I think making this explicit would be help users differentiate between `onefuzz_url`  and `multi_tenant_domain`.